src/storage/segcache/src/common/mod.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// item info masks and shifts
+pub const TAG_MASK: u64 = 0xFFF0_0000_0000_0000;
```

Comment:
Some ASCII art here would greatly enhance the reviewer's experience.

Reply:
Added - not sure if you have any examples off the top of your head that address masks better.